awx/ui_next/src/screens/Template/WorkflowJobTemplate.jsx

```diff
@@ -110,6 +132,10 @@ class WorkflowJobTemplate extends Component {
                 <WorkflowJobTemplateDetail
                   hasTemplateLoading={hasContentLoading}
                   template={template}
+                  webHookKey={webHookKey}
+                  onSetContentLoading={() =>
```

Comment:
I plan to remove this in favor of allowing the Detail View to handle it's own content loading state.